rifServices/src/main/java/org/sahsu/rif/services/graphics/RIFMapsParameters.java

```diff
@@ -291,32 +298,68 @@ public RIFMapsParameter getRIFMapsParameter(String key) {
 	/**
 	 * Setup default map parameters
 	 */
-	private void setupDefaultMapParameters() 
+	private void setupDefaultMapParameters(final SQLManager manager, final CachedRowSetImpl rif40Studies) 
 		throws Exception {
 		
-		RIFMapsParameter rifMapsParameter1 = new RIFMapsParameter(	
-			"relative_risk"		/* resultsColumn */,
-			"quantile"		/* Classifier function name */, 
-			"PuOr"			/* colorbrewer palette: http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=8 */, 
-			9				/* numberOfBreaks */, 
-			true			/* invert */);
-		rifMapsParameters.put("viewermap", rifMapsParameter1);
-				
-		RIFMapsParameter rifMapsParameter2 = new RIFMapsParameter(
-			"smoothed_smr"		/* resultsColumn */,
-			"quantile"		/* Classifier function name */, 
-			"PuOr"			/* colorbrewer palette: http://colorbrewer2.org/#type=diverging&scheme=PuOr&n=8 */, 
-			9				/* numberOfBreaks */, 
-			true			/* invert */);		
-		rifMapsParameters.put("diseasemap1", rifMapsParameter2);
+		String studyID=manager.getColumnFromResultSet(rif40Studies, "study_id");
+		String selectStateText=manager.getColumnFromResultSet(rif40Studies, "select_state", true /* allowNulls */, false /*  allowNoRows */);
+		boolean isDiseaseMappingStudy=true;
+		
+		if (selectStateText != null) {
+			JSONObject selectStateJson = new JSONObject(selectStateText); // Check it parses OK
+			String studyTypeStr = selectStateJson.optString("studyType");
+			if (studyTypeStr != null && studyTypeStr.equals("risk_analysis_study")) {
+				isDiseaseMappingStudy=false;
+				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; use database select state study type: risk analysis"); 
+				this.studyType = StudyType.RISK_ANALYSIS;
+			}
+			else if (studyType != null && studyType.equals("disease_mapping_study")) {
```

Comment:
I don't think this can ever be true: an enum can't equal a String. Unless the enum's `toString` method is called implicitly, but it would be clearer just to compare against the enum value.

Reply:
```if (studyTypeStr != null && studyTypeStr.equals("risk_analysis_study")) {``` is comparing strings. Replaced with rifConcepts studyType code:
```
		if (selectStateText != null) {
			JSONObject selectStateJson = new JSONObject(selectStateText); // Check it parses OK
			String studyTypeStr = selectStateJson.optString("studyType");
			if (studyTypeStr != null && studyTypeStr.equals(StudyType.RISK_ANALYSIS.type())) {
				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; use database select state study type: risk analysis"); 
				this.studyType = StudyType.RISK_ANALYSIS;
			}
			else if (studyType != null && studyType.equals(StudyType.DISEASE_MAPPING.type())) {
				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; use database select state study type: disease mapping"); 
				this.studyType = StudyType.DISEASE_MAPPING;
			}
			else {
				rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; no database select state study type, assume: disease_mapping;" +
					lineSeparator +
					"; selectStateJson: " + selectStateJson.toString(2)); 
			}	
		}
		else {
			rifLogger.info(getClass(), "rif40_studies.study_id: " + studyID + "; no database select state, assume study type: disease_mapping"); 
		}
```
Tested OK (before and now) for both DM and RA, i.e. produces the right maps